src/collections/_documentation/workflow/integrations/integration-platform/ui-components.md

```diff
@@ -216,6 +238,68 @@ For a more complete description of the grammar, see the [source code](https://gi
     level="info"
 %}
 
+
+# Create/Link Issue
+
+If you have a UI component for issue linking and a user attempts to create or link an issue, it will send a request to your service based on the `uri` you provide.
+
+## Request Format
+
+The request that is sent is a `POST` request so all the information is stored in the body.
+
+**Schema**
+
+```
+{
+    "fields": <Object>,
+    "installationId": <String>,
+    "issueId": <String>,
+    "webUrl": <String>,
+    "project": {"slug": <String>, "id": <String>},
+    "actor": {"type": "user", "name": <String>, "id": <String>},
+}
+```
+
+**Attributes**
+
+- `fields` - Key/value pairs of the data in the form fields.
+- `installationId` - The ID of the installation associated with the request.
+- `issueId` - The ID of the issue in Sentry
+- `webUrl` - The URL of the issue in Sentry
+- `project`:
+   - `slug` - The slug of the project in Sentry
+   - `id` - The ID of the project in Sentry 
+- `actor`:
+   - `name` - The name of the user in Sentry
+   - `id` - The ID of user in Sentry 
+
+
+
+## Response Format
+
+When creating or linking an issue, the response format *must* have the following schema:
+
+
+**Schema**
+
+```
+{
+    "webUrl": <String>,
+    "project": <String>,
+    "identifier": <String>,
+}
+```
+
+
+**Attributes**
+
+- `webUrl`: The URL of the linked issue in your project management system.
+- `project`: The first part of the displayed issue that should probably be associated with a project in your system.
+- `identifier`: The second part displayed issue name that should be unique the project.
+
+If the display name for your issue is `Front_End#12`, the project would be `Front_End` and the identifier would be `12`.
```

Comment:
```suggestion
If the display name for your issue is `Front_End#12`, the project would be `Front_End`, and the identifier would be `12`.
```

Reply:
I think maybe a screenshot could be useful here to show where in the app this is getting used. Also I think that maybe this should go right under **Attributes** to make more sense. 


Also I think we should add more of a description to make it clearer: 

When an external issue is created or linked in Sentry, Sentry shows a display name that links back to the service where it was either created or linked. The display name is composed of two pieces, the `project`,  and the `identifier`, connected by a `#`. (then maybe show the example)

@MimiDumpling Im sure this could be worded better but this is what i got so far lol

Reply:
How about this:

"The display name is composed of two pieces: the project and the identifier. A hash (#) connects each piece."

Reply:
@MimiDumpling updated: 

![Screen Shot 2020-03-31 at 10 51 33 AM](https://user-images.githubusercontent.com/8533851/78058758-a06e3d80-733d-11ea-87e2-4fd3ef8b5906.png)

Reply:
This part was fine so I left it out of my suggestion. Feel free to use it:

When an external issue is created or linked in Sentry, Sentry shows a display name that links back to the service where it was either created or linked.

Reply:
@MimiDumpling Got it, I put that back in

Reply:
Thank you!